src/EnergyPlus/DataZoneEquipment.cc

```diff
@@ -1418,6 +1421,8 @@ namespace DataZoneEquipment {
 		} else if ( ( SELECT_CASE_var == OAFlowSum ) || ( SELECT_CASE_var == OAFlowMax ) ) {
 			// Use sum or max of per person and the following
 			DSOAFlowPerZone = OARequirements( DSOAPtr ).OAFlowPerZone;
+			double FA = Zone(ActualZoneNum).FloorArea;
```

Comment:
We don't use `double`.  Long ago we created our own typedef called `Real64` so all floating point variables in E+ should be `Real64`.